shell/platform/common/cpp/public/flutter_texture_registrar.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_COMMON_CPP_PUBLIC_FLUTTER_TEXTURE_REGISTRAR_H_
+#define FLUTTER_SHELL_PLATFORM_COMMON_CPP_PUBLIC_FLUTTER_TEXTURE_REGISTRAR_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "flutter_export.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+// Opaque reference to a texture registrar.
+typedef struct FlutterDesktopTextureRegistrar*
```

Comment:
Add a
```
struct FlutterDesktopTextureRegistrar;
```

line above this, so the struct is declared. (This is needed to avoid warnings in some compilation modes when nothing else declares the struct later, as is the case for Windows in this iteration since no such struct ever exists.)